ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2339,32 +2339,31 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: baremetal-lab-upi
-- as: gcp-ipi-baselinecaps-none-additionalcaps-f28
-  cron: 29 16 29 * *
+- as: gcp-ipi-baselinecaps-none-additionalcaps-f7
+  cron: 49 5 5,13,21,28 * *
   steps:
     cluster_profile: gcp-qe
     env:
-      ADDITIONAL_ENABLED_CAPABILITIES: baremetal marketplace openshift-samples CSISnapshot
-        Console Insights Storage NodeTuning MachineAPI
       BASELINE_CAPABILITY_SET: None
       E2E_RUN_TAGS: '@amd64 and @gcp-ipi and @network-ovnkubernetes and not @fips'
+      PERSISTENT_MONITORING: "false"
       TAG_VERSION: '@4.14'
     test:
     - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-gcp-ipi
-- as: gcp-ipi-baselinecaps-none-additionalcaps-f28-destructive
-  cron: 31 22 5 * *
+    workflow: cucushift-installer-rehearse-gcp-ipi-capability-addtionalcaps
+- as: gcp-ipi-baselinecaps-none-additionalcaps-f7-destructive
```

Comment:
Destructive test with f7 ?

Reply:
this job is to test disable optional capability, select one randomly from full set, since fullset contains about 10 caps on 4.14, to make sure all items could be tested before 4.14 GA, adjust frequency to f7 on all jobs.
we can adjust frequency after 4.14 GA.

Reply:
Maybe we can keep destruct jobs on f28, and others on f7, once that, that means, we can have 10 ci outptut, GA date is the end of Oct, I think it should be okay.

Reply:
make sense, change back to f28